Jenkinsfile

```
@@ -73,10 +73,12 @@ pipeline {
         GOFLAGS = '-mod=readonly'
       }
       steps {
-        withGithubNotify(context: 'Lint') {
-          withBeatsEnv(archive: false, id: 'lint') {
+        withGithubNotify(context: "Lint: 02") {
+          withBeatsEnv(archive: false, id: "lint-02") {
```

Are these names intended?
```suggestion
        withGithubNotify(context: "Lint") {
          withBeatsEnv(archive: false, id: "lint") {
```

---

yep, was a test I have to change it to fix the GitHub checks